mobile/justfile

```diff
@@ -0,0 +1,74 @@
+# To use this file, install Just: cargo install just
+line_length := "100"
+
+default: gen
+precommit: gen lint
+
+# deps: Install missing dependencies.
+deps: deps-gen deps-android deps-ios
+
+deps-gen:
+	cargo install flutter_rust_bridge_codegen
+
+# deps-android: Install dependencies for Android (build targets and cargo-ndk)
+deps-android:
+	cargo install cargo-ndk
+	rustup target add aarch64-linux-android armv7-linux-androideabi x86_64-linux-android i686-linux-android
+
+# deps-ios: Install dependencies for iOS
+deps-ios:
+	cargo install cargo-lipo
+	rustup target add aarch64-apple-ios x86_64-apple-ios
+
+gen:
+    flutter pub get
+    flutter_rust_bridge_codegen \
+        --rust-input native/src/api.rs \
+        --dart-output lib/bridge_generated.dart \
+        --c-output ios/Runner/bridge_generated.h \
+        --dart-decl-output lib/bridge_definitions.dart \
+        --dart-format-line-length {{line_length}}
+
+#        --c-output macos/Runner/bridge_generated.h \
+
+native:
+    cd native && cargo build
+
+# Build Rust library for Android native targets
+android:
+	cd native && cargo ndk -o ../android/app/src/main/jniLibs build
+
+# ios: Build Rust library for iOS
+ios:
+	cd native && cargo lipo
+	cp native/target/universal/debug/libnative.a ios/Runner
+
+run:
+    flutter run
+
+clean:
+    flutter clean
+    cd native && cargo clean
+    
+serve *args='':
```

Comment:
Are we aiming for a web target? if not, I would remove the serve target. if not we should add 

```
deps-web:
  cargo install wasm-pack
```

However, tested locally and it did not work for me.